proprietary/tokens/build.js

```diff
@@ -28,7 +28,7 @@ function generateSharedConfig(mode) {
           destination: `${name}.theme.css`,
           format: 'css/variables',
           options: {
-            selector: '.amsterdam-theme',
+            selector: `.amsterdam-theme${name === 'index' ? '' : `--${name}`}`,
```

Comment:
This is a stylistic choice: should we prefer a logical conjuncation over the ternary operator in compact cases like this?

```suggestion
            selector: `.amsterdam-theme${name !== 'index' && `--${name}`}`,
```

It’s a bit more compact, not difficult to read, and we already use it often.

Reply:
> This is a stylistic choice

Not quite. AND returns a boolean value, so this would result in `.amsterdam-themefalse`

Reply:
Hah, I already doubted whether I wasn’t making a point that opposed my other comment